File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/coupon/exception/SupplyLessThenMinimumException.java
package band.gosrock.domain.domains.coupon.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class SupplyLessThenMinimumException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new SupplyLessThenMinimumException();

    private SupplyLessThenMinimumException() {
        super(CouponErrorCode.SUPPLY_LESS_THEN_MINIMUM);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/coupon/exception/SupplyLessThenMinimumException.kt
package band.gosrock.domain.domains.coupon.exception

import band.gosrock.common.exception.DuDoongCodeException

class SupplyLessThenMinimumException private constructor() : DuDoongCodeException(CouponErrorCode.SUPPLY_LESS_THEN_MINIMUM) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = SupplyLessThenMinimumException()
    }
}
